graph_def_editor/visualization/graphviz_style.py

```diff
@@ -0,0 +1,33 @@
+"""Default GraphViz styles to use."""
```

Comment:
Can you please add a copyright header to the top of this file? You can copy it from any other file in the project.